base/abstractarray.jl

```diff
@@ -1113,7 +1113,7 @@ end
 """
     parent(A)
 
-Returns the "parent array" of an array view type (e.g., `SubArray`), or the array itself if
+Return the "parent array" of an array view (e.g. a `SubArray`), or the array itself if
```

Comment:
Since this applies to many types, perhaps worth mentioning one more?
```suggestion
Return the "parent array" of an array view (e.g. a `SubArray`, or a `Transpose`), or the array itself if
```

Reply:
Good idea, but `Transpose` is in `LinearAlgebra`. It would then have to be `LinearAlgebra.Transpose`, or something else in `Base`, like `ReinterpretArray` ?

Reply:
Perhaps it can be re-worded to mention `view` & `transpose`, since these are the user-facing functions? 
```
Return the "parent array” which was the argument of `view`, `transpose` or 
another function which creates a wrapper (like `SubArray`, 
`ReshapedArray` or `LinearAlgebra.Transpose`). Or else return the array 
itself if it cannot be further unwrapped.
```
I’m not sure that’s better!